test/OpenTelemetry.Contrib.Shared.Tests/SqlProcessorTestCases.cs

```diff
@@ -15,36 +18,68 @@ public static class SqlProcessorTestCases
         Converters = { new JsonStringEnumConverter() },
     };
 
-    public static IEnumerable<object[]> GetTestCases()
+    private static readonly Uri Uri = new("https://raw.githubusercontent.com/open-telemetry/semantic-conventions/refs/heads/main/docs/non-normative/database-test-cases/db-sql-test-cases.json");
```

Comment:
I think it would be better to keep the existing in-repo copy read from resources, then in a follow-up I can look to add a GitHub Actions workflow to periodically open a PR to update the content if it drifts. Then we're isolated from CI breaking if some drastic refactoring happens in open-telemetry/semantic-conventions.

We can then also avoid the use of HttpClient and the `#if`s to deal with calling it sync in the `[MemberData]` method.